flixel/animation/FlxAnimationController.hx

```diff
@@ -777,6 +777,55 @@ class FlxAnimationController implements IFlxDestroyable
 		play(AnimName);
 		return AnimName;
 	}
+		
+	public function animationList():Array<FlxAnimation>
+	{
+		var arrayNames:Array<FlxAnimation> = [];
+		
+		for (i in _animations)
```

Comment:
suggestion: I try to avoid `i` except for traditional `for (i in 0...length)` loops. I recommend `anim`

Reply:
Yeah, I changed the name of the things so it's resolved